frontend/src/pages/VR_Campus/VR_Campus.jsx

```diff
@@ -0,0 +1,125 @@
+import Img from 'gatsby-image';
+
+function fixName(name) {
+    name = name.replace('--', ': ');
+    name = name.replace('-', ' ');
+    return name
+        .trim()
+        .toLowerCase()
+        .replace(/\w\S*/g, w => w.replace(/^\w/, c => c.toUpperCase()));
+}
+
+const VR_Campus = () => (
+    <StaticQuery
+        query={graphql`
+            {
+                allFile(
+                    filter: { relativeDirectory: { eq: "vr-photo-spheres" } }
+                ) {
+                    edges {
+                        node {
+                            name
+                            publicURL
+                            relativePath
+                            childImageSharp {
+                                fixed(width: 300, height: 200) {
+                                    src
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        `}
+        render={({ allFile: { edges } }) => (
+            <div>
+                <div
+                    style={{
+                        margin: '2rem auto',
+                        maxWidth: '910px',
+                        textAlign: 'center'
+                    }}
+                >
+                    <Title variant="h5" gutterBottom className="title">
+                        {translate('VR Campus')}
+                    </Title>
+                    <br />
+
+                    <Typography>
+                        {translate(
+                            "Welcome to IEEE uOtttawa's virtual campus photo spheres."
+                        )}
+                    </Typography>
+                    <br />
+                    {/** CSS Grid used here because I was having problems with Material Grid */}
+                    <div
+                        style={{
+                            display: 'inline-grid',
+                            gridTemplateColumns: 'repeat(auto-fit, 300px)',
+                            justifyContent: 'center',
+                            width: '100%'
+                        }}
+                    >
+                        {edges.map(({ node, i }) => {
+                            // Every node name delineates the category. It should be
+                            // sorted before this point so the categories are in order.
+                            return (
+                                <div
+                                    style={{
+                                        padding: '4px',
+                                        position: 'relative'
+                                    }}
+                                    key={i}
+                                >
+                                    <a href={`/vr/${node.name}`}>
+                                        <img
+                                            src={node.childImageSharp.fixed.src}
+                                            alt={`/vr/${node.name}`}
```

Comment:
the alt should be a description of what the image represents. if there's no way to dynamically generate that, you can just leave it as an empty string.

Reply:
I think this is fine. He is dynamically generating the names

https://user-images.githubusercontent.com/26887757/97791588-72d6c480-1baa-11eb-90d9-5bee0df3a124.png

Reply:
It would be better to put the transformed title here, if anything.

Reply:
Yea I'd prefer that